query_executor.go

```diff
@@ -23,7 +23,7 @@ func (q *queryExecutor) attemptQuery(qry ExecutableQuery, conn *Conn) *Iter {
 	iter := qry.execute(conn)
 	end := time.Now()
 
-	qry.attempt(q.pool.keyspace, end, start, iter)
+	qry.attempt(q.pool.keyspace, end, start, iter, conn.Address())
```

Comment:
can we just use the host info instead here?

Reply:
unless I'm missing something, I can access the `HostInfo` from the `Conn` only by using the `localHostInfo()` method no ?
Method that can return an error, not ideal

Reply:
the *HostInfo is now accessible on *conn